src/Umbraco.Core/EmbeddedResources/Lang/da.xml

```diff
@@ -2134,7 +2134,7 @@ Mange hilsner fra Umbraco robotten
     <key alias="deleteThisSearch">Slet denne søgning</key>
     <key alias="findLogsWithRequestId">Find logs med request Id</key>
     <key alias="findLogsWithNamespace">Find logs med Namespace</key>
-    <key alias="findLogsWithMachineName">Find logs med maskin navn</key>
+    <key alias="findLogsWithMachineName">Find logs med maskinnavn</key>
```

Comment:
This is the literal translation, so probably right. I would just never call it *maskine*  in Danish 🤔

Reply:
I actually don't remember changing that, but I agree about _maskine_ :D (I don't really know what _machine_ refers to in the original - is it something like the server's ID?)

Reply:
This is the name of the computer/server. In .NET you can call `Environment.MachineName` to get the computer name, so following that, *maskinnavn* is correct. I just wouldn't call it that in Danish.